src/app/search/loadouts/loadout-filter-types.ts

```diff
@@ -23,5 +27,9 @@ export interface LoadoutFilterContext {
  */
 export interface LoadoutSuggestionsContext {
   loadouts?: Loadout[];
+  /**
+   * The selected store on the loadouts page
+   */
+  selectedLoadoutsStore?: DimStore;
```

Comment:
I had to make this optional due to the filter help component. I could probably play around with it to try and make it be able to be defined, but not sure if this is already factored into the design.

```
ERROR in src/app/search/FilterHelp.tsx:108:61
TS2345: Argument of type 'LoadoutSuggestionsContext | SuggestionsContext' is not assignable to parameter of type 'LoadoutSuggestionsContext'.
```

Reply:
Probably just need to plumb the types through a bit more. I can mess with it later.